content/tutorials/reactor-event-ifttt.textile

```diff
@@ -0,0 +1,164 @@
+---
+title: "Reactor Events Tutorial: Integrating your Ably channel with IfThisThenThat"
+alt_title: "Ably-IFTTT integration"
+excerpt: "Learn how to use our Reactor Functions to trigger IFTTT events"
+section: tutorials
+category: external-services
+index: 100
+platform: browser
+authors:
+- author_name: Jo Franchetti
+  author_bio: "Developer advocate at ably"
+  author_profile_url: https://github.com/thisisjofrank
+  author_image: ://avatars2.githubusercontent.com/u/3490640?s=460&v=4
+level: medium
+reading_time: 10
+languages:
+  - javascript
+  - nodejs
+ably_product: reactor
+tags:
+  - ably-features
+---
+
+The "Ably Reactor":https://www.ably.io/reactor provides a way to trigger events and to stream data from Ably's pub/sub channels. This tutorial will go over how to use the Ably Reactor with the "IfThisThenThat Webhooks service":https://ifttt.com/maker_webhooks. IfThisThenThat (IFTTT) allows you to create chains of conditional statements called Applets. These Applets can be triggered by evebts on your Ably Channel. They in turn can trigger other web services such as email, social media, chat apps, IoT devices etc.
+In this specific example we'll be using IFTTT to trigger a post to a "Slack channel":https://ifttt.com/slack every time a message is published on your "Ably channel":https://www.ably.io/channels.
+
+h2. What we'll be building
+
+We'll be building a bridge simulator game using HTML, CSS, JavaScript and "Reactor Functions":https://www.ably.io/documentation/general/functions. Players will receive commands from their "captain" via Slack. These commands will let them know what to press in the game to move to the next command. Clicking the controls that match the commands will win them the game, mistakes will lose and crash the spaceship.
+
+<a href="/images/tutorials/reactor/reactor-event-ifttt-game.png" target="_blank">
+  <img src="/images/tutorials/reactor/reactor-event-ifttt-game.png" style="width: 100%" alt="screenshot of the game we will be making">
+</a>
+
+h2. Prerequisite Accounts
+
+You'll need an Ably account, login or sign up for free at "https://www.ably.io/":https://www.ably.io/.
+You'll also need an IFTTT account, login or sign up at "https://ifttt.com/":https://ifttt.com/.
+Finally you'll need an account with Slack and a workspace to send messages to, sign in or create a workspace at "https://slack.com/signin":https://slack.com/signin
+
+<%= partial 'tutorials/_step-1-setup-free-account' %>
+
+<%= partial 'tutorials/_create-reactor-rule-event.textile' %>
+
+Choose the "*IFTTT*" service integration option
+
+On the resulting form we'll need an IFTTT key to connect Ably to IFTTT with webhooks. 
+
+# Click the "open IFTTT Webhook Settings" button (this will take you to the IFTTT web app)
```

Comment:
It'd be good if we can keep how we're highlighting various bits of text consistent for the tutorial. For example, for 'click on X', do you want to bold + quotes around it, just bold, just quotes, etc. Happy whatever way you'd prefer, just good to keep it the same for everything.